rmgpy/rmg/model.py

```diff
@@ -1199,8 +1199,12 @@ def prune(self, reactionSystems, toleranceKeepInEdge, maximumEdgeSpecies, minSpe
                 pruneDueToRateCounter += 1
             # Keep removing species with the lowest rates until we are below the maximum edge species size
             elif numPrunableSpecies - len(speciesToPrune) > maximumEdgeSpecies:
-                logging.info('Pruning species {0} to make numEdgeSpecies smaller than maximumEdgeSpecies'.format(spec)) # repeated ~15 lines below
-                speciesToPrune.append((index, spec))
+                if not maxEdgeSpeciesRateRatios[index] < toleranceMoveToCore:
```

Comment:
Perhaps `>` or `>=` instead of `not <`?
Or was the `not` left here by mistake?

Reply:
This should be fixed now.